rmax:                     100 #maximum radius of points to consider
num_maxbinpoints:         200 #max Number of Points in each Bin
num_seedpoints:           10  #Number of seed points to use in GP
num_ang_bins:             72  #Number of angular bins to use
num_lin_bins:             200 #Number of linear bins to use

# GP Kernel hyperparameters
gp_lengthparameter:       10
gp_covariancescale:       1
gp_modelnoise:            0.3
gp_groundmodelconfidence: 5
gp_grounddataconfidence:  5

gp_groundthreshold:       0.3 #Ground Height Threshold
robotheight:              1.2 #Used for drivable surface detections I think
seeding_maxrange:         50  #Maximum range for seed points
seeding_maxheight:        15  #Maximum height for seed points
